### DISCUSSION TITLE
RabbitMQ 3.8.11 Is communication possible between each node in the state of network partitioning?

### DISCUSSION THREAD

**Discussion by lcrlim** [rabbitmq/rabbitmq-server#10006] [Questions]
When a mirrored cluster consisting of two nodes (N1, N2) enters a network partitioned state, with a publisher (P1) and consumer (C1) connected to N1, and another publisher (P2) and consumer (C2) connected to N2, it is possible for a message published by P1 to be received by C2. Is this behavior considered normal?

**Comment by michaelklishin** [comment]
@lrclim RabbitMQ 3.8 has [reached end of life in 2022](https://rabbitmq.com/versions.html) and classic mirrored queues [have long been deprecated](https://www.rabbitmq.com/migrate-mcq-to-qq.html).

Two node clusters are [explicitly recommended against](https://www.rabbitmq.com/clustering.html#node-count).

Publishers do not communicate with consumers. They communicate with queue leader replicas, then that replica communicates with consumers. Depending on the number of messages "in flight" (received by the node), the partition handling strategy used and [failure detection timing](https://rabbitmq.com/nettick.html), it may sometimes seem like two nodes on two sides of a split are communicating.
